### ISSUE TITLE
Going Deeper with Convolutions

### ISSUE THREAD

**Issue by NoahHalonen** [NoahHalonen/NoahHalonen-Capstone_Resources#11]
### Title
Going Deeper with Convolutions

### URL
https://arxiv.org/abs/1409.4842v1

### Summary 
This paper introduces a deep convolutional neural network architecture named Inception, which set new records for classification and detection in the ImageNet Large-Scale Visual Recognition Challenge 2014 (ILSVRC14). The Inception architecture efficiently utilizes computing resources within the network, allowing for increased depth and width of the network without a corresponding increase in computational cost. This efficiency is achieved through a design that incorporates multi-scale processing insights and the Hebbian principle. The architecture, known as GoogLeNet in its specific incarnation for ILSVRC14, significantly outperforms previous state-of-the-art models while using fewer parameters.

### Key Points 
- Introduced the Inception architecture, which optimizes network depth and width under a constant computational budget.
- Achieved state-of-the-art results in the ILSVRC 2014 competition with GoogLeNet, a 22-layer deep network.
- Utilized Hebbian principles and multi-scale processing to inform architectural decisions.
- Despite its depth, GoogLeNet uses 12x fewer parameters than its predecessors, demonstrating efficient use of computational resources.
- The paper discusses various aspects of training deep networks, including the importance of dropout, data augmentation, and methods to ensure gradient propagation through the deep network.
- Also highlighted are the auxiliary classifiers added to intermediate layers to aid in training and provide regularization.
- The architecture's design principles allow for scalable and efficient deep neural networks suitable for real-world applications, even on devices with limited computational resources.

### Citation
@misc{szegedy2014going,
      title={Going Deeper with Convolutions}, 
      author={Christian Szegedy and Wei Liu and Yangqing Jia and Pierre Sermanet and Scott Reed and Dragomir Anguelov and Dumitru Erhan and Vincent Vanhoucke and Andrew Rabinovich},
      year={2014},
      eprint={1409.4842},
      archivePrefix={arXiv},
      primaryClass={cs.CV}
}
